forum/qa-plugin/report-reason/script.js

```diff
@@ -0,0 +1,95 @@
+$(document).ready(function()
+{
+    $(".qa-form-light-button-flag").attr("type", "button");
+    
+    $(".qa-form-light-button-flag").click( function()
+    {
+        var postId = $(this).data("postid");
+        var postType = $(this).data("posttype");
+        var parentId = $(this).data("parentid");
+        
+        $("#flagbox-popup").show();
+        
+        $(".qa-flag-reasons-wrap .closer").click( function()
+        {
+            $("#flagbox-popup").hide();
+        });
+        
+        // focus on first element, then Enter and Escape key work
+        $('.qa-flag-reasons-wrap input').first().focus();
```

Comment:
Czy możesz wyjaśnić gdzie w tej linijce (albo w innej) jest "Enter and Escape key work"? Bo z samego kodu można wyczytać, że focusujesz pierwszy element. :) Toteż komentarz jest raczej zbędny.

Reply:
Ja jsa nie ruszałem. Został taki jaki był ;)

Reply:
Spoko, ale jeśli chcesz go dodać do kodu Forum, to moim zdaniem powinien być lepiej napisany.